### PR TITLE
Add Created field to UserResource

### PR DESCRIPTION
**Pull request by iryan2**
Related to this [PR](https://github.com/OctopusDeploy/OctopusDeploy/pull/12437)

### CODE REVIEW

**Review by andrewabest — CHANGES_REQUESTED** [review]
I suspect you're still missing an approval test that needs approving for netfx

File: source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt [hunk 1]
```diff
@@ -5459,6 +5459,7 @@ Octopus.Client.Model
   {
     .ctor()
     Boolean CanPasswordBeEdited { get; set; }
+    DateTimeOffset Created { get; set; }
```
Comment (iryan2) [inline source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt:5462]: @andrewabest Should this be `protected set;` be be consistent with [this comment](https://github.com/OctopusDeploy/OctopusDeploy/pull/12437#discussion_r898505299)?

**Comment by iryan2** [comment]
> I suspect you're still missing an approval test that needs approving for netfx

I'm not sure I follow, the checks on this PR passed and the tests are green locally: 

image

What makes you suspect that?

**Comment by iryan2** [comment]
> I suspect you're still missing an approval test that needs approving for netfx

@andrewabest Oh, was this comment intended for [this PR](https://github.com/OctopusDeploy/OctopusDeploy/pull/12654) instead?

File: source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt (hunk 1 above)
Comment (andrewabest) [reply source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt:5462]: No, not in the Octopus.Clients repository.

**Comment by andrewabest** [comment]
> I'm not sure I follow, the checks on this PR passed and the tests are green locally:

There is a [netfx approval file](https://github.com/OctopusDeploy/OctopusClients/blob/master/source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETFramework.approved.txt) too - not sure why the build has gone green if this hasn't been updated - there may be a build issue if we are not running the client tests on netframework. One for @slewis74 to look into at some point. For this PR, you can just update the netframework approval file with the same change applied to the netcore file.

**Comment by iryan2** [comment]
> > I'm not sure I follow, the checks on this PR passed and the tests are green locally:
> 
> There is a [netfx approval file](https://github.com/OctopusDeploy/OctopusClients/blob/master/source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETFramework.approved.txt) too - not sure why the build has gone green if this hasn't been updated - there may be a build issue if we are not running the client tests on netframework. One for @slewis74 to look into at some point. For this PR, you can just update the netframework approval file with the same change applied to the netcore file.

Ahh thanks, done.

**Review by andrewabest — APPROVED** [review]
Looks good

**Pull request merged**